config/_default/menus/main.en.yaml

```diff
@@ -3742,11 +3742,16 @@ menu:
       parent: code_analysis
       identifier: code_analysis_github_prs
       weight: 4
+    - name: IDE Plugins
+      url: developers/ide_plugins
+      parent: code_analysis
+      identifier: code_analysis_ide_plugins
+      weight: 5
```

Comment:
Have you tried a [full-page reuse](https://datadoghq.atlassian.net/wiki/spaces/docs4docs/pages/1182958054/Markdown#Reusing-a-full-page) here? Might be a good use case for it.

Reply:
Thanks for the suggestions! I think I'll create some dedicated pages in the Code Analysis doc set that are tailored to Code Analysis-specific interactions with the IDE plugins.